src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/EditAndContinue/EditAndContinueProvider.cs

```diff
@@ -81,7 +81,7 @@ public int GetExceptionSpanCount(out uint pcExceptionSpan)
 
         public int GetExceptionSpans(uint celt, ENC_EXCEPTION_SPAN[] rgelt, ref uint pceltFetched)
         {
-            uint pceltFetchedResult = 0;
+            uint pceltFetchedResult = pceltFetched;
```

Comment:
You can lose the local var here and forward the ref directly.

Reply:
Might behave differently in case of exception.

Reply:
I can't lose the local, can't capture a ref parameter in a lambda.